src/pykeen/models/init.py

```diff
@@ -31,7 +31,8 @@ def embedding_xavier_uniform_(embedding: nn.Embedding, gain: float = 1.) -> nn.E
     In the following example, an embedding is initialized using the suggested gain for the rectified
     linear unit (ReLu).
 
-    >>> e = nn.Embedding(num_embeddings=3, embedding_dim=5)
+    >>> import pykeen.utils
+    >>> e = pykeen.utils.Embedding(num_embeddings=3, embedding_dim=5)
```

Comment:
should we have a `pykeen.nn` module so it seems more like a drop-in replacement?

Reply:
Sure, it might not be well-placed in `utils` anyway

Reply:
Done in aeda9b89